pkg/chains/formats/intotoite6/intotoite6_test.go

```diff
@@ -122,9 +136,12 @@ func TestTaskRunCreatePayload1(t *testing.T) {
 	if err != nil {
 		t.Errorf("unexpected error: %s", err.Error())
 	}
-	if diff := cmp.Diff(expected, got); diff != "" {
+	// Ignore Materials since their order can vary and result in flakes
```

Comment:
You can use https://pkg.go.dev/github.com/google/go-cmp/cmp/cmpopts#SortSlices to implement the same thing in the same diff.

Reply:
Thanks! this is much cleaner